gauge-tests/env/chrome/browser.properties

```diff
@@ -0,0 +1,2 @@
+browser = chrome
+webdriver.chrome.driver =D:/WebDriver/chromedriver/chromedriver23.exe
```

Comment:
paths... can we avoid that?

Reply:
i will put a note there, couldn't find how to run gauge with env variables, it is some witchcraft there.